apps/state_mediator/test/state_mediator/mqtt_mediator_test.exs

```diff
@@ -55,6 +55,16 @@ defmodule StateMediator.MqttMediatorTest do
 
     test "are sent to the state module" do
       {:ok, _pid} = start_link(@opts)
+
+      {:ok, client} = EmqttFailover.Connection.start_link(configs: [@opts[:url]])
+      Process.sleep(500)
```

Comment:
question: why do you need this?

Reply:
Which part? We need a client to publish a message, `test.mqtt.org` seems to do this automatically whenever a client connects. We need `Process.sleep` because `Connection.start_link` is asynchronous and `publish` will fail if we don't pause first, because the client is not actually connected.

Reply:
It can send a message when connected: https://hexdocs.pm/emqtt_failover/EmqttFailover.ConnectionHandler.Parent.html (example: https://github.com/mbta/concentrate/blob/a478d5fc96db2d259974cdc86516850a4fa8a713/lib/concentrate/sink/mqtt.ex)

Reply:
Updated with `EmqttFailover.ConnectionHandler.Parent`.